src/OrbitBase/include/OrbitBase/ThreadPool.h

```diff
@@ -68,9 +68,13 @@ class ThreadPool : public orbit_base::Executor {
   // Action in a different way than simply calling Action::Execute. This can for
   // example be used to run some operation before and/or after the original
   // Action.
-  static std::shared_ptr<ThreadPool> Create(
+  [[nodiscard]] static std::shared_ptr<ThreadPool> Create(
       size_t thread_pool_min_size, size_t thread_pool_max_size, absl::Duration thread_ttl,
       std::function<void(const std::unique_ptr<Action>&)> run_action = nullptr);
+
+  // Globally accessible ThreadPool with a fixed number of threads that equal to or slightly smaller
+  // than the number of available cores on the system.
+  [[nodiscard]] static std::shared_ptr<ThreadPool> GetGlobalWorkerThreadPool();
```

Comment:
Not sure if this maybe should go into the `Executor` call because ideally we don't require a thread pool. Any executor suffices. Also internally we call this function `DefaultExecutor()`, but I don't feel strongly about this.

Also it should probably return a raw pointer because it should be non-owning. This also works better with `TaskGroup` as it shouldn't own the executor.

Reply:
Made it return a pointer and renamed to ThreadPool::GetDefaultThreadPool(). I left it in ThreadPool as I feel it's more descriptive and it allows to create a default thread pool in the "GetDefaultThreadPool()" method.